react/tic-tac-toe/src/app/screens/Login/components/LoginInput/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import styles from './styles.module.scss';
+
+function LoginInput({ input, label, type, meta: { error, submitFailed } }) {
+  return (
+    <div>
+      <label>{label}</label>
+      <input
+        {...input}
+        placeholder={label}
+        type={type}
+        className={styles.loginInput}
+      />
+      {submitFailed && error && <span>{error}</span>}
+    </div>
```

Comment:
Really good job here. It's better to have this type of components like this so you can re use them. Just one suggestion, if you are not using this div to hold a class you can just use `<> .... </>`

Reply:
yes i code so good than its even better than the one at the redux-forms tutorial 🤣